app/src/main/res/layout/account_preference.xml

```diff
@@ -54,7 +54,9 @@
             android:layout_alignStart="@id/displayName"
             android:maxLines="4"
             android:text="@string/preferences_account_default_name"
-            android:textColor="?primaryText" />
+            android:textColor="?primaryText"
+            android:fontFamily="@font/metropolis_bold"
+            android:letterSpacing="0.05" />
```

Comment:
[This comment](https://github.com/mozilla-mobile/fenix/pull/6986#issuecomment-596852375) seems to suggest that we shouldn't need any custom letter spacing anymore.  Is this one a special case, or was it just accidentally missed?